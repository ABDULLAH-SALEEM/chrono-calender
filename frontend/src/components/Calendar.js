import React from "react";
const Calendar = () => <div>Calendar Component</div>;
export default Calendar;
