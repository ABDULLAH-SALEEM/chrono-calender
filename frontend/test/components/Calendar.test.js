import React from "react";
import { render, screen } from "@testing-library/react";
import Calendar from "../../src/components/Calendar";

describe("Calendar", () => {
  it("renders without crashing", () => {
    render(<Calendar />);
    expect(screen.getByText(/Calendar Component/i)).toBeInTheDocument();
  });

  it("matches snapshot", () => {
    const { asFragment } = render(<Calendar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
