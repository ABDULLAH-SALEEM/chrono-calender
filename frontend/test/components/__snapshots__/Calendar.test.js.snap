// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar matches snapshot 1`] = `
<DocumentFragment>
  <div>
    Calendar Component
  </div>
</DocumentFragment>
`;
